bin/faucet/src/server/resources/index.html

```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miden Faucet</title>
     <link rel="stylesheet" href="./index.css">
+    <!-- Load SHA3 library directly -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/js-sha3/0.8.0/sha3.min.js" integrity="sha384-NKbaJKqJPrFeb86qfU2iNy4KO3oyIe6F5xm1PBYMpAkiVTxHjjafUKlebp0mAeNC" crossorigin="anonymous"></script>
```

Comment:
I guess maybe this is the reason for using SHA3? I'm assuming BLAKE3 option is not as easily available?

Reply:
Replied in the previous comment: https://github.com/0xMiden/miden-node/pull/831#discussion_r2085212818